sympy/printing/dot.py

```diff
@@ -7,27 +7,32 @@
 from sympy.core.compatibility import default_sort_key
 from sympy.core.add import Add
 from sympy.core.mul import Mul
+from sympy.printing.repr import srepr
 
 __all__ = ['dotprint']
 
 default_styles = ((Basic, {'color': 'blue', 'shape': 'ellipse'}),
           (Expr,  {'color': 'black'}))
 
 
-sort_classes = (Add, Mul)
```

Comment:
Why was `x**2` sorted then? Maybe a test should be added for it.

Reply:
Here is `dotprint(2**x+x**3)`:

![image](https://user-images.githubusercontent.com/90703/57267097-74bad880-7044-11e9-8566-3981cfa685c7.png)